docs/docs/section-2/step-01.md

```diff
@@ -64,16 +64,16 @@ Before starting, ensure you have:
 Navigate to the `section-2/step-01` directory and start the application:
 
 === "Linux / macOS"
-    ```bash
-    cd section-2/step-01
-    ./mvnw quarkus:dev
-    ```
+```bash
+cd section-2/step-01
+./mvnw quarkus:dev
+```
 
 === "Windows"
-    ```cmd
-    cd section-2\step-01
-    mvnw quarkus:dev
-    ```
+```cmd
+cd section-2\step-01
+mvnw quarkus:dev
+```
```

Comment:
This is a regression, you're breaking the tabs. I fixed it in my previous changes.